dotNetInstallerToolsLib/FileUtil.cpp

```diff
@@ -221,15 +221,19 @@ DVLib::FileVersion DVLib::wstring2fileversion(const std::wstring& version)
 {
     FileVersion result = { 0, 0, 0, 0 };
 
+    bool const hasLeadingV = !version.empty() && (version[0] == 'v' || version[0] == 'V');

```

Comment:
I think it would be simpler to strip the leading `v` if it's there, and then just leave the parsing code alone. Also more future proof when some other magical format needs to be supported. WDYT?

Reply:
The code would be much simpler that way. That was my first idea. However, that means making a copy of the string and I though of avoiding that. However, it's probably not an important performance issue and the code would be much cleaner that way.

Reply:
I simplified the handling of the leading `v` and updated docs. Also incremented the minor version.

Reply:
This function is called rarely enough not to worry about copying a string ;)